spec/javascript/support/puppeteer/steps/supplementalApplicationSteps.js

```diff
@@ -1,43 +1,36 @@
+import {
+  selectSubstatusIfRequired,
+  checkForStatusUpdateSuccess,
+  commentInputSelector,
+  openStatusModalSelector,
+  submitStatusModalSelector,
+  unselectedStatusMenuItem
+} from './leaseUpStatusSteps'
 import sharedSteps from './sharedSteps'
 
 const testStatusModalUpdate = async (page) => {
```

Comment:
This seems weird to totally outsource the expectations and test to another file but 🤷. Instead of totally refactoring I just updated it to use the shared functions + selectors I created